erdpy_network/proxy_network_provider_test.py

```diff
@@ -106,3 +108,63 @@ def test_get_sc_invoking_tx(self):
         assert result.is_completed == True
         assert len(result.contract_results.items) > 0
         assert result.data == 'issue@54455354@54455354@03e8@00@63616e4d696e74@74727565@63616e4275726e@74727565@63616e4368616e67654f776e6572@74727565@63616e55706772616465@74727565'
+
+    def test_send_transactions(self):
+        first_tx = FirstTransaction()
+        second_tx = SecondTransaction()
+        third_tx = ThirdTransaction()
+
+        transactions = [first_tx, second_tx, third_tx]
+
+        expected_hashes = [
+            "6e2fa63ea02937f00d7549f3e4eb9af241e4ac13027aa65a5300816163626c01",
+            "37d7e84313a5baea2a61c6ab10bb29b52bc54f7ac9e3918a9faeb1e08f42081c"
+        ]
+
+        assert self.proxy.send_transaction(transactions[0]) == expected_hashes[0]
+        assert self.proxy.send_transactions(transactions) == (2, {'0': f"{expected_hashes[0]}", '1': f"{expected_hashes[1]}"})
+
+
+class FirstTransaction:
```

Comment:
Instead of having 3 classes, we can also define a single one, such as `DummyTransaction`, which receives a dictionary in the constructor - simply to be returned by `to_dictionary()`.